maven/spec/dependabot/maven/requirement_spec.rb

```diff
@@ -16,7 +16,7 @@
     context "with a pre-release version" do
       let(:requirement_string) { "1.3.alpha" }
 
-      it { is_expected.to be_satisfied_by(Gem::Version.new("1.3.a")) }
+      it { is_expected.to be_satisfied_by(Gem::Version.new("1.3.alpha")) }
```

Comment:
This test is checking for an exact match. In the previous maven standard 1.3.alpha was considered the same as 1.3.a but based on the real maven standard this is not correct.